cli/src/base.ts

```diff
@@ -88,10 +88,14 @@ export abstract class BaseCommand extends Command {
     const profile = apiKey ? undefined : await getProfile();
 
     apiKey = apiKey || profile?.apiKey;
-    if (!apiKey)
-      this.error(
-        'Could not instantiate Xata client. No API key found. Please run `xata auth login` or configure a project with `xata init`.'
-      );
+    if (!apiKey) {
+      this.error('Could not instantiate Xata client. No API key found.', {
```

Comment:
Should this error? Or should it call the login command directly?

Reply:
I think erroring is fine.

Reply:
Agreed, I wouldn't call another command at least without asking. The `suggestions` helper looks really nice in `this.error`.